internal/disk/disk.go

```diff
@@ -15,19 +15,17 @@ import (
 )
 
 type PartitionTable struct {
-	// Size of the disk.
-	Size uint64
-	UUID string
-	// Partition table type, e.g. dos, gpt.
-	Type       string
+	Size       uint64 // Size of the disk (in bytes).
+	UUID       string // Unique identifier of the partition table (GPT only).
+	Type       string // Partition table type, e.g. dos, gpt.
 	Partitions []Partition
 }
 
 type Partition struct {
-	Start    uint64
-	Size     uint64
-	Type     string
-	Bootable bool
+	Start    uint64 // Start of the partition in sectors
+	Size     uint64 // Size of the partition in sectors
+	Type     string // `gpt` or `dos`
```

Comment:
Why do both the partitions and the partition table have a `Type` field? Will they ever differ?

Reply:
I am not sure that is right, actually. Let me check

Reply:
No, the comment is wrong, this is not `gpt` or `dos` but the partition type, Linux data, EFI, etcpp

Reply:
Ah, got it. Makes sense then.